python/common/org/python/types/Object.java

```diff
@@ -552,14 +552,6 @@ public void __delitem__(org.python.Object index) {
         throw new org.python.exceptions.AttributeError(this, "__contains__");
     }
```

Comment:
I'm not sure that this AttributeError is strictly necessary -- seems like a `__contains__` call is either successful (it's defined elsewhere) or a `TypeError` is thrown (and this is constructed after this `AttributeError` is thrown and caught). Could this be replaced by `org.python.types.NotImplemented`?